src/components/common/data-not-found/data-not-found.directive.coffee

```diff
@@ -0,0 +1,63 @@
+module = angular.module('impac.components.common.data-not-found',[])
+
+module.directive('commonDataNotFound', ($templateCache, $log, $http, ImpacAssets, ImpacTheming, ImpacMainSvc) ->
+  return {
+    restrict: 'A',
+    scope: {
+      isKpi: '=?'
```

Comment:
Yes I agree with this. It was isKpi as kpi image capabilities where developed at one point in the history of this branch.